partiql-parser/src/partiql.pest

```diff
@@ -2,18 +2,102 @@ WHITESPACE = _{ " " | "\t" | "\x0B" | "\x0C" | "\r" | "\n" }
 
 COMMENT = _{ LineComment | BlockComment }
 
-BlockComment = { "/*" ~ (BlockComment | (!"*/" ~ ANY))* ~ "*/" }
-LineComment = { "--" ~ (!"\n" ~ ANY )* ~ "\n" }
+BlockComment    = _{ "/*" ~ (BlockComment | (!"*/" ~ ANY))* ~ "*/" }
+LineComment     = _{ "--" ~ (!"\n" ~ ANY )* ~ "\n" }
 
 // TODO implement a full grammar, this is a very primitive version to start
 //      working with Pest and its APIs.
 
-// Entry point for full query parsing
-Query = _{ SOI ~ Token+ ~ EOI}
+//
+// Entry point for full query parsing.
+//
+Query = _{ SOI ~ Expression ~ EOI}
+
+//
+// Expressions
+//
+
+Expression = _{
+      SelectExpression
+    | ValueExpression
+}
+
+// TODO finish the SELECT statement
+
+SelectExpression = {
+    Select ~ (From ~ (Where)?)?
```

Comment:
So `From` is optional here? In partiql-lang-kotlin, a `FROM` clause is currently required, but there was an enhancement [issue](https://github.com/partiql/partiql-lang-kotlin/issues/210) to make it optional. From the [spec](https://partiql.org/assets/PartiQL-Specification.pdf#page=9), `FROM` seems required in the BNF grammar.

Reply:
technically, `from ` should be optional in the Kotlin implementation IMHO this is optional in other SQLs

Reply:
Yes, although non-standard, we should make it optional.